### PR TITLE
Update how-to-run-agent.md

### PR DESCRIPTION
**Pull request by ValOlson**

### CODE REVIEW

File: docs/agent-reference/how-to-run-agent.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Running the Device Update for IoT Hub Reference Agent
 
+## Required Dependencies
```
Comment (nihemstr) [inline docs/agent-reference/how-to-run-agent.md:3]: > Required Dependencies [](http://example.com/codeflow?start=2&length=22)

Recommend we add steps for adding the Microsoft Repository and how to install deliveryoptimization using 
``` sudo apt-get install deliveryoptimization

like many of our documents point out to do.

**Review by nihemstr — APPROVED** [review]
:shipit:

**Review by nihemstr — CHANGES_REQUESTED** [review]
:clock1:

File: docs/agent-reference/how-to-run-agent.md (hunk 1 above)
Comment (ValOlson) [reply docs/agent-reference/how-to-run-agent.md:3]: Nic and I are following up Jimmy, meanwhile added: 

Alternatively you can add the Microsoft Repository to your local APT repositories using these (instructions)[https://docs.microsoft.com/windows-server/administration/linux-package-repository-for-microsoft-software]. Then you will be able to install the necessary DO dependencies using <bash script to install DO>

**Review by jimson-msft — CHANGES_REQUESTED** [review]
:clock1:

File: docs/agent-reference/how-to-run-agent.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+
+* Install the [Delivery Optimization SDK](https://github.com/microsoft/do-client)
+* Install the [Delivery Optimization components](https://github.com/microsoft/do-client#installing-do-client-components) i.e., the Delivery Optimization agent & Delivery Optimization APT plugin 
+
+Alternatively you can add the Microsoft Repository to your local APT repositories using these (instructions)[https://docs.microsoft.com/windows-server/administration/linux-package-repository-for-microsoft-software]. Then you will be able to install the necessary DO dependencies using
+
+```bash
+sudo apt-get install deliveryoptimization-plugin-apt
+```
+
```
Comment (jimson-msft) [inline docs/agent-reference/how-to-run-agent.md:13]: This is only true for Ubuntu 18.04 amd64.

For these set of docs, which are instructing users how to build and install DU + DO from source. I would remove any mention of packages.

File: docs/agent-reference/how-to-run-agent.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 # Running the Device Update for IoT Hub Reference Agent
 
+## Required Dependencies
+
+1. Build the Device Update agent and its dependancies using these [instructions](https://github.com/Azure/iot-hub-device-update/blob/main/docs/agent-reference/how-to-build-agent-code.md)
+2.  Follow the instructions from [this](https://github.com/microsoft/do-client#delivery-optimization-client) page to build and install the Delivery Optimization agent and plug-in. Note: Delivery Optimization SDK is already installed as part of step 1 above.
```
Comment (jimson-msft) [inline docs/agent-reference/how-to-run-agent.md:6]: >  [](http://example.com/codeflow?start=3&length=1)

extra space

**Review by jimson-msft — APPROVED** [review]
:shipit:

File: docs/agent-reference/how-to-run-agent.md (hunk 2 above)
Comment (ValOlson) [reply docs/agent-reference/how-to-run-agent.md:7]: Updated to:
Follow the instructions from this page to build and install the Delivery Optimization agent and plug-in. Note: Delivery Optimization SDK is already installed as part of step 1 above.

**Pull request merged**